src/openforms/registrations/contrib/objects_api/models.py

```diff
@@ -25,34 +25,52 @@ def get_payment_status_update_text() -> str:
 
 class ObjectsAPIConfig(SingletonModel):
     """
-    global configuration and defaults
+    Entry point to the default Objects APIs set.
     """
 
-    objects_service = models.OneToOneField(
+    default_objects_api_group = models.ForeignKey(
+        to="ObjectsAPIGroupConfig",
+        on_delete=models.PROTECT,
+        verbose_name=_("default Objects API group."),
+        help_text=_("Which Objects API group should be used by default."),
+        null=True,
+    )
```

Comment:
see earlier remarks - probably best to only keep legacy config here and then drop that too in Open Forms 3.0